packages/select/theme/material/vaadin-select-styles.js

```diff
@@ -27,6 +27,13 @@ const select = css`
     opacity: 1;
   }
 
+  :host([has-label]:not([focused]):not([invalid]):not([theme='always-float-label']))
```

Comment:
Corresponding style in `input-field-shared.js`: https://github.com/vaadin/web-components/blob/3e0117fbdfce63c81ede6e70772c8b3bb2c0fb2a/packages/vaadin-material-styles/mixins/input-field-shared.js#L107